db/migrate/20170217085547_add_type_to_cloud_object_store_container.rb

```diff
@@ -0,0 +1,5 @@
+class AddTypeToCloudObjectStoreContainer < ActiveRecord::Migration[5.0]
+  def change
+    add_column :cloud_object_store_containers, :type, :string
```

Comment:
I can think of two lookups:
a) by `type` only
```ruby
ManageIQ::Providers::Amazon::StorageManager::S3::CloudObjectStoreContainer.all
```

b) by `type` and `ems_id`
```ruby
ManageIQ::Providers::Amazon::StorageManager::S3.cloud_object_store_containers
```

Would you agree?